packages/instrument-utils/src/storedInstrument/ManageInstrumentsModal/ManageInstrumentsModal.tsx

```diff
@@ -96,6 +102,17 @@ class ManageInstrumentsModal extends Component<
         const cardInstruments = instruments.filter(isCardInstrument);
         const bankInstruments = instruments.filter(isBankAccountInstrument);
         const accountInstruments = instruments.filter(isAccountInstrument);
+        const braintreeAchInstrument = instruments.filter(isBraintreeAchInstrument);
```

Comment:
Maybe we can remove Braintree word in global (core) packages? I don't really like the idea, that we are adding provider specific logic to generic packages..